### ISSUE TITLE
NFS Timeout in advanced settings no longer works

### ISSUE THREAD

**Issue by rwkjones** [xbmc/xbmc#23238]
### Describe the bug
Library updates timeout after a very long time when one of my NFS servers is down. In fact most operations including exiting seem to stall or take minutes to complete.

I notice this error in the debug log:
2023-05-02 15:09:57.033 T:1045665 warning <general>: nfstimeout unsupported

## Expected Behavior
NFS operations time out after 5 seconds

## Actual Behavior
NFS operations time out after minutes

### To Reproduce
1. Create the advanced settings file with nfstimeout
2.
3.

### Debuglog

The debuglog can be found here:
https://paste.kodi.tv/aqitexesos.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [x] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version:
Kodi Media Center 20.1 (20.1.0) Git:20230312-289ec664e3

**Comment by rwkjones** [comment]
Suit yourselves...

**Comment by rwkjones** [comment]
So what's wrong with the bug report, that made the computer say no? The error in my debug log suggests very strongly that support for "nfstimeout" in advancedsettings.xml has changed. I asked for help on the forums, no reply, hence posting here. I completely appreciate this is a community supported effort - I have been a linux sysadmin for 20 years, and a massive fan of Kodi since Eden. If the answer's "Well, we dropped support for it" fine, but It would be nice to know so I can try to work around it :-)

**Comment by wsnipex** [comment]
it should still be supported, unless the libnfs version is too old: https://github.com/xbmc/xbmc/blob/master/xbmc/settings/AdvancedSettings.cpp#L533-L537
Is this an official kodi build? Can you try with a v21 nightly?

**Comment by rwkjones** [comment]
Yep, the nightly build fixed it, there was nfstimeout error the log file and NFS timeouts took roughly five seconds
